packages/page-statistics/src/index.tsx

```diff
@@ -2,13 +2,14 @@ import React, { FC } from 'react';
 import styled from 'styled-components';
 
 import { DepositStatistics } from './DepositStatistics';
+import { DKGEggnetStatistics } from './DKGEggnetStatistics';
 import { DKGStatistics } from './DKGStatistics';
 import { RelayersStatistics } from './RelayerStatistics';
 import { StatisticsOverview } from './StatisticsOverview';
 import { WithdrawalStatistics } from './WithdrawalStatistics';
 
 type StatisticsPageProps = {
-  view: string;
+  view: 'overview' | 'deposits' | 'withdrawals' | 'relayers' | 'dkg' | 'dkg-eggnet';
 };
 
 const PageStatistics: FC<StatisticsPageProps> = ({ view }) => {
```

Comment:
Can we use switch here for clarity, and ts would till if there is unhandled branch